test/com/google/javascript/jscomp/Es6RewriteBlockScopedDeclarationTest.java

```diff
@@ -860,7 +860,7 @@ public void testDoWhileForOfCapturedLet() {
   }
 
   // https://github.com/google/closure-compiler/issues/1124
```

Comment:
#1124 is unrelated to this fix. I added a separate test case for #1157 down below.